test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.OrderUsingHtmlHelpers.Encoded.html

```diff
@@ -1,11 +1,10 @@
-﻿<html>
+<html>
 <head>
     <meta charset="utf-8" />
     <title></title>
 </head>
 <body>
-<form action="HtmlEncode[[/UrlEncode[[HtmlGeneration_Order]]/UrlEncode[[Submit]]]]" method="HtmlEncode[[post]]">
-        <div>
+<form action="HtmlEncode[[/UrlEncode[[HtmlGeneration_Order]]/UrlEncode[[Submit]]]]" method="HtmlEncode[[post]]">        <div>
```

Comment:
This looks like a bug in the Razor change. There's no C# between the `<form>` and `<div>` elements and there _is_ whitespace. Is Razor now getting confused about transitions into and out of tag helpers?

Source is
``` html
    <form asp-controller="HtmlGeneration_Order" asp-action="Submit" asp-antiforgery=" true">
        <div>
```